src/app/campaign/store/effects/advertiser.effects.ts

```diff
@@ -10,18 +10,24 @@ export class AdvertiserEffects {
   loadAdvertisers$ = createEffect(() =>
     this.actions$.pipe(
       ofType(advertiserActions.AdvertisersLoad),
-      switchMap(() => this.advertiserService.loadAdvertisers()),
-      map(docs => advertiserActions.AdvertisersLoadSuccess({ docs })),
-      catchError(error => of(advertiserActions.AdvertisersLoadFailure({ error })))
+      switchMap(() =>
```

Comment:
Probably my fault for doing it this way with flattened streams initially all over in the Effects. Error handling in Effects should be nested within a higher order operator or the effect stream will complete.  https://ngrx.io/guide/effects#handling-errors You can see in the marble diagrams where I took out the `(a|)` (action+completion grouping) in the tests that the effect will not complete on error, which is what we want. Ngrx 9 changed effects to only retry a default of 10 times when it resubscribes on error, so it seemed best to get these right. https://ngrx.io/guide/migration/v9#limit-retries-to-10-by-default